torchio/transforms/preprocessing/spatial/crop_or_pad.py

```diff
@@ -102,12 +102,12 @@ def _compute_center_crop_or_pad(self, sample: dict):
         diff_shape = target_shape - source_shape
 
         cropping = -np.minimum(diff_shape, 0)
-        if cropping.any():
-            cropping_params = self._get_six_bounds_parameters(cropping)
+        #if cropping.any():
+        cropping_params = self._get_six_bounds_parameters(cropping) if cropping.any() else None
 
         padding = np.maximum(diff_shape, 0)
-        if padding.any():
-            padding_params = self._get_six_bounds_parameters(padding)
+        #if padding.any():
```

Comment:
I added a default value `None` if no padding or cropping is needed. This raised up a bug during my testing. There might be a cleaner, more generic way to handle this ?

Reply:
This looks ok to me.